python/servo/build_commands.py

```diff
@@ -318,6 +319,16 @@ def build(self, release=False, dev=False, jobs=None, params=None, media_stack=No
                 "lib", "pkgconfig"
             )
 
+        if 'windows' in host:
+            process = subprocess.Popen('("%s" %s > nul) && "python" -c "import os; print(repr(os.environ))"' %
+                                       (os.path.join(vs_dirs['vcdir'], "Auxiliary", "Build", "vcvarsall.bat"), "x64"),
+                                       stdout=subprocess.PIPE, shell=True)
+            stdout, _ = process.communicate()
+            exitcode = process.wait()
+            encoding = locale.getpreferredencoding()  # See https://stackoverflow.com/a/9228117
+            if exitcode == 0:
+                os.environ.update(eval(stdout.decode(encoding)))
```

Comment:
Does this actually do anything? All of the calls below use `env` with the call, so `os.environ` doesn't actually get used. Am I missing something?

I have my VS installation on `E:` and this causes the build to be unable to find msvcrt.lib (and other core libs)